packages/frontend/stalker-app/src/app/modules/jobs/subscriptions/subscription-templates.ts

```diff
@@ -26,7 +26,7 @@ job:
 conditions:
   - lhs: string
     operator: contains
-    rhs: ring
+    rhs: string
```

Comment:
It was intended, it is an example of a `true` condition, but I see how it could look like a typo. A non-issue that should disappear with #237

Reply:
omg my bad 😆